cachelib/allocator/CacheAllocatorConfig.h

```diff
@@ -597,7 +630,8 @@ CacheAllocatorConfig<T>& CacheAllocatorConfig<T>::setCacheName(
 }
 
 template <typename T>
-CacheAllocatorConfig<T>& CacheAllocatorConfig<T>::setCacheSize(size_t _size) {
+CacheAllocatorConfig<T>& CacheAllocatorConfig<T>::setCacheSizeImpl(
```

Comment:
Shall we not change `setCacheSize` function as other users are using this function without setting the multi-tier cache config?  Can we add a new setCacheSize function only for multi-tier caches?